readthedocs/builds/tasks.py

```diff
@@ -361,7 +361,10 @@ def send_build_status(build_pk, commit, status, link_to_build=False):
     :param status: build status failed, pending, or success to be sent.
     """
     # TODO: Send build status for BitBucket.
-    build = Build.objects.get(pk=build_pk)
+    build = Build.objects.filter(pk=build_pk).first()
+    if not build:
+        return
```

Comment:
Should we log here? Is this actually an error, or some weird outcome of a race condition?

Reply:
This is probably when a project has been deleted, since we no longer delete builds when a version is deleted. Or if we are using a replica db, that may be too, but don't think we are.